lmctl/cli/commands/__init__.py

```diff
@@ -25,6 +25,8 @@
 from .infrastructure_key import *
 from .intents import *
 from .processes import *
+from .permissiontype import *
 from .resourcedriver import *
 from .resource_managers import *
-from .resource_packages import *
\ No newline at end of file
+from .resource_packages import *
+from .objectgroups import *
```

Comment:
to follow convention, files named `objectgroups` should be named `object_groups` instead

Reply:
Done